src/admin/redux/modules/email/reducer.js

```diff
@@ -97,7 +97,7 @@ const reducer : Reducer<State, Action>
 
 export default reducer
 
-const sort_by_date = r_compose(reverse, sortBy(prop('created_at')), map(over(lensProp('created_at'), Date.parse)))
+const sort_by_date = pipe([map(over(lensProp('created_at'), Date.parse)), sortBy(prop('created_at')), reverse])
```

Comment:
eslint with very few rules is what I am doing at the moment. I do think you have to install like 4 modules because of React, JSX etc. As you say I am not sure it's really worth it. Also tbh, I do enjoy the pruning, probably a bit too much 😊